test/libyul/yulOptimizerTests/fullInliner/multi_fun_callback.yul

```diff
@@ -36,20 +36,15 @@
 //     function f(x)
```

Comment:
The order functions are processed changed here: Now, first `g` is inlined into `f`. Then we take a look at `g` and do not inline the call to `f` anymore because it is directly recursive now.

Reply:
Is the comment at the beginning of this file still relevant?
If it is, there is a typo there `statemenst`, might wanna fix that in this PR if the comment stays.

Reply:
I think it is still relevant.